### DISCUSSION TITLE
MongoDB - updating fields across multiple Collections?

### DISCUSSION THREAD

**Discussion by thadguidry** [9tigerio/db2rest#524] [Ideas]
Hmm, we currently only have support of updating a single collection at a time.
We could change `String collectionName` to `Array collections`, or something like that?
Then we handle by running multiple Update statements, 1 for each collection in the Array against the criteria to patch (update)?

**Comment by kdhrubo** [comment]
Closing as no dev to be done on Mongodb going forward. Support ends 2.0
